FSharp-4.5/FS-1053-span.md

```diff
@@ -264,21 +264,34 @@ let F1() =
     c.InstanceProperty <- today.AddDays(2.0)
 ```
 
+#### Scoping for byrefs
 
-#### Implicit address-of when calling members with parameter type `inref<'T>` 
+The general idea is that a let-bound value cannot have its reference escape the scope from which it was declared, effectively its visibility. For example:
```

Comment:
What does "effectively its visibility" mean?

Reply:
The visibility scope of the value.

Reply:
Okay, I think this makes sense as-is.